### PR TITLE
Changes notation for NonTraditionalFPlane

### PR DESCRIPTION
**Pull request by glwagner**
This PR changes the names of the `y` and `z` components of the rotation vector for the non-traditional f-plane to `fy` and `fz`.

It updates the tests and documentation. It also adds a section about the non-traditional f-plane approximation to the "physics" section of the documentation.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -106,26 +106,26 @@ closures = (
...
+        #include("test_solvers.jl")
+        #include("test_pressure_solvers.jl")
+        #include("test_coriolis.jl")
+        #include("test_buoyancy.jl")
+        #include("test_surface_waves.jl")
+        #include("test_models.jl")
+        #include("test_time_stepping.jl")
+        #include("test_boundary_conditions.jl")
+        #include("test_forcings.jl")
+        #include("test_turbulence_closures.jl")
```
Comment (suyashbire1) [inline test/runtests.jl]: Enable all of these again.

File: docs/src/physics/coriolis_forces.md [hunk 2]
```diff
@@ -2,23 +2,40 @@
 
 The Coriolis model controls the manifestation of the term $\bm{f} \times \bm{u}$ in the momentum equation.
 
-## The $f$-plane approximation
+## The traditional and non-traditional $f$-plane approximation
 
 Under an $f$-plane approximation[^3] the reference frame in which
-the momentum and tracer equations are are solved rotates at a constant rate around a
-vertical axis, such that
+the momentum and tracer equations are are solved rotates at a constant rate.
```
Comment (suyashbire1) [inline docs/src/physics/coriolis_forces.md]: multiple "are"

File: docs/src/physics/coriolis_forces.md [hunk 3]
```diff
@@ -2,23 +2,40 @@
...
+      directions east, north, and up is
       ```math
-          \bm{f} \approx \frac{4 \pi}{\text{day}} \sin \varphi \bm{\hat z} \, , $
+          \bm{f} \approx \frac{4 \pi}{\text{day}} \left ( \cos \varphi \bm{\hat y} + \sin \varphi \bm{\hat z} \right ) \, , $
       ```
-      where $\phi$ is latitude and the Earth's rotation rate is approximately $2 \pi / \text{day}$.
-      This approximation neglects the vertical component of Earth's rotation vector at $\varphi$.
+      where the Earth's rotation rate is approximately $2 \pi / \text{day}$.
+      The *traditional* $f$-plane approximation neglects the $y$-component of this projection, which is appropriate for 
+      fluid motions with small horizontal-to-vertical aspect ratios.
```
Comment (suyashbire1) [inline docs/src/physics/coriolis_forces.md]: I think this should read "large horizontal-to-vertical aspect ratios"

File: test/runtests.jl (hunk 1 above)
Comment (glwagner) [reply test/runtests.jl]: doh!

File: docs/src/physics/coriolis_forces.md (hunk 3 above)
Comment (glwagner) [reply docs/src/physics/coriolis_forces.md]: hmm yes, common mistake from me...

**Comment by ali-ramadhan** [comment]
Weirdly enough Travis got stuck for 18 hours and GitLab CI timed out so I restarted both builds.

**Pull request merged**